gammapy/image/models/tests/test_models.py

```diff
@@ -74,11 +87,22 @@ def test_skyshell2D():
     desired = 0.0002976757280439522 / u.deg ** 2
```

Comment:
@cdeil The test value changed quite a bit here, you wrote the old ``TemplateModel`` class. Any ideas why this is?